types/bootstrap3-dialog/index.d.ts

```diff
@@ -0,0 +1,195 @@
+// Type definitions for bootstrap3-dialog 1.35
```

Comment:
I'd suggest enclosing these interfaces on their namespace with the same name as your variable declaration: BootstrapDialog

Reply:
I like the namespace idea. I could then shorten some of the type names from `BootstrapDialog.BootstrapDialogContext` to `BootstrapDialog.DialogContext`.

What do you think?

Reply:
I think it's the best as well.